### PR TITLE
fix(max_nonce): is -1 for BESU because stored as a (signed) long

### LINKED ISSUE #1277: MAX_NONCE is -1 for Besu
RefTest 121 - `d12g0v0_London`

### PR DESCRIPTION
**Pull request by letypequividelespoubelles**

### CODE REVIEW

File: arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/signals/AbortingConditions.java [hunk 1]
```diff
@@ -90,7 +88,8 @@ public void prepare(Hub hub) {
             final Address myAddress = hub.currentFrame().accountAddress();
             final long creatorNonce =
                 hub.messageFrame().getWorldUpdater().get(myAddress).getNonce();
-            yield Long.compareUnsigned(EIP2681_MAX_NONCE, creatorNonce) < 0;
+            // The nonce is a (signed) long for BESU so EIP2681_MAX_NONCE == -1
```
Comment (letypequividelespoubelles) [inline arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/signals/AbortingConditions.java:91]: one of the two change

File: arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/Hub.java
Comment (letypequividelespoubelles) [inline arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/Hub.java:1]: only moving function to begin with all BESU hook functiun.

File: arithmetization/src/main/java/net/consensys/linea/zktracer/module/oob/OobOperation.java [hunk 2]
```diff
@@ -269,16 +269,16 @@ private void populateColumnsForEvmInstruction(MessageFrame frame) {
...
+        final boolean hasCode = deployedAccount != null && deployedAccount.hasCode();
 
         final CreateOobCall createOobCall = (CreateOobCall) oobCall;
         createOobCall.setValue(EWord.of(frame.getStackItem(0)));
         createOobCall.setBalance(creatorAccount.getBalance().toUnsignedBigInteger());
         createOobCall.setNonce(BigInteger.valueOf(nonce));
         createOobCall.setHasCode(hasCode);
         createOobCall.setCallStackDepth(BigInteger.valueOf(frame.getDepth()));
-        createOobCall.setCreatorNonce(BigInteger.valueOf(creatorAccount.getNonce()));
+        createOobCall.setCreatorNonce(longToUnsignedBigInteger(creatorAccount.getNonce()));
```
Comment (letypequividelespoubelles) [inline arithmetization/src/main/java/net/consensys/linea/zktracer/module/oob/OobOperation.java:281]: second change

File: linea-constraints
Comment (letypequividelespoubelles) [inline linea-constraints:1]: just an update to latest master

**Review by OlivierBBB — APPROVED** [review]
LGTM

**Pull request merged**